src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -1656,6 +1656,120 @@ public function get_test_https_status() {
 		return $result;
 	}
 
+	/**
+	 * Tests if Full Page Cache is available.
```

Comment:
A bit of a nit-pick, but you sometimes use "Page Cache" and other times "Page Caching". We should align on one way. I'm not sure which one is the more appropriate one, can any native English speakers help out here with feedback?

Also no need to capitalize "Full Page Cache", it should just be regular English words (lowercase).

Reply:
@westonruter Any thoughts here?

Reply:
To be honest we pretty much used Page Cache and Page Caching when implementing this in the AMP plugin. Either I think is preferable to “Advanced Cache” as used in WordPress core. I suppose “Full Page Cache” would be better than “Full Page Caching”, as _cache_ is used instead of _caching_ for the Object Cache.

Reply:
Should we tick with Full Page Cache then?

Reply:
Yeah, let's go with "full page cache" / "Full Page Cache".

Reply:
```suggestion
	 * Tests if a full page cache is available.
```